tests/run-pass/cmath.rs

```diff
@@ -0,0 +1,13 @@
+fn cbrt(f: f64) -> f64 {
+    f.cbrt()
+}
+
+fn hypot(f1: f64, f2: f64) -> f64 {
+    f1.hypot(f2)
+}
+
+fn main() {
+    assert_eq!(cbrt(4.3), 1.6261333316791686);
```

Comment:
I guess you can round both values or use one of the "almost eq" crates for float assertions

Reply:
Well, not really the crates, but inlining that shouldn't be too hard, even without `fabs`.
```rust
fn almost_eq(x: f64, y: f64) -> bool {
    const EPSILON: f64 = 1e-20;
    let diff = x-y;
    diff < EPSILON && diff > -EPSILON
}
```